### PR TITLE
Blog post for ESDS event homework

### PR DESCRIPTION
**Pull request by jukent**
This post will have set up instructions and links to the homework for each of the session break outs, for that I will need the links to content picked out by @hCraker @dcherian and @rmshkv.

@anissa111 I believe you were putting together the environment file for the tutorial, please update this PR or coordinate with me to make the instructions better match what you have. :)

### CODE REVIEW

**Comment by anissa111** [comment]
@jukent yes, sort of! The environment file is being updated as needed in the geocat-tutorials repo [here](https://github.com/NCAR/geocat-tutorials/blob/main/environment.yml)

**Comment by jukent** [comment]
> @jukent yes, sort of! The environment file is being updated as needed in the geocat-tutorials repo [here](https://github.com/NCAR/geocat-tutorials/blob/main/environment.yml)

Great thanks! We'll want participants to fork and clone the repo? I'll update the instructions today.

**Comment by anissa111** [comment]
Oh no they definitely don't have to do that. We're recommending they use the binder link in the readme. There's instructions for the different methods of viewing the content in the readme.

File: posts/2022/esds-event-prep.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+## Session homework
+
+The events on Day 1 will follow a flipped classroom model, where you watch some tutorials and do study on your own before coming together in your breakout group to troubleshoot, ask questions, or dive deeper. To get the most out of these sessions, please complete the prep work selected by each session leader.
+
+The breakout groups are:
+
+- [Python Basics](https://docs.google.com/document/d/1ku15eYYGFujpGaiabJCDL4Lo8WBJilG5x2GbjMsjsNU/edit?usp=sharing)
+- [Numpy](https://docs.google.com/document/d/1fCLZaJwQxiV12QHAwVuu5fHv5h6Atoq9xT_rdnEX3OM/edit?usp=sharing)
+- [Visualization]()
```
Comment (rmshkv) [inline posts/2022/esds-event-prep.md:36]: Here's the link to my homework doc

https://docs.google.com/document/d/1pCZNh1NToWETMBwHQr-rj12l-ABJQpdEXK8-IGa6r4E/edit?usp=sharing

File: posts/2022/esds-event-prep.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+- [Xarray]()
```
Comment (dcherian) [inline posts/2022/esds-event-prep.md:37]:
```suggestion
- [Xarray](https://docs.google.com/document/d/12mNISjqjhuYaf2MN9USbwqWz-Z3pGKk89yYrXkCy5No/edit#heading=h.2f5s7kmk7pxa)
```

File: posts/2022/esds-event-prep.md (hunk 1 above)
Comment (jukent) [reply posts/2022/esds-event-prep.md:36]: Great thanks!

**Comment by hCraker** [comment]
@dcherian @jukent I need to request access to see the xarray homework document. Can we change the permissions so that anyone with the link can view it?

File: posts/2022/esds-event-prep.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+### Local Install
+
+For those of you who prefer to work on your local machine, please install Miniconda and Jupyter before the ESDS event.
+
+You can follow the [Project Pythia instructions to install Python on Jupyter](https://foundations.projectpythia.org/foundations/jupyter.html).
+
+If you have trouble installing miniconda - try to uninstall and reinstall it, making sure you select the correct install location.
+
+Once installed, you will
```
Comment (erogluorhan) [inline posts/2022/esds-event-prep.md:30]: Would you like to replace the following instructions with a referral to https://github.com/NCAR/geocat-tutorials/blob/main/README.md as it already gives local config instructions?

Comment (jukent) [reply posts/2022/esds-event-prep.md]: Done!

**Review by erogluorhan — APPROVED** [review]
Looks great!

**Pull request merged**